docs/site-generator/Makefile

```diff
@@ -0,0 +1,155 @@
+# Default shell
```

Comment:
@alcortesm we'll take a look into migrating the script to Python in the future. Right now, we'll stick with this strange chimera, because it is a lot of code to migrate and this needs to be working as soon as possible. At the start we did not think it would become that big of a Makefile.